core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -407,7 +474,13 @@ private[spark] object SparkConf extends Logging {
         "The spark.cache.class property is no longer being used! Specify storage levels using " +
         "the RDD.persist() method instead."),
       DeprecatedConfig("spark.yarn.user.classpath.first", "1.3",
-        "Please use spark.{driver,executor}.userClassPathFirst instead."))
+        "Please use spark.{driver,executor}.userClassPathFirst instead."),
+      DeprecatedConfig("spark.kryoserializer.buffer.mb", "1.4",
```

Comment:
I wanted to still provide this message to clarify that fractional values are no longer allowed.
Thank you,
Ilya Ganelin


From: Marcelo Vanzin <notifications@github.com<mailto:notifications@github.com>>
Reply-To: apache/spark <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, April 24, 2015 at 10:16 AM
To: apache/spark <spark@noreply.github.com<mailto:spark@noreply.github.com>>
Cc: "Ganelin, Ilya" <ilya.ganelin@capitalone.com<mailto:ilya.ganelin@capitalone.com>>
Subject: Re: [spark] [SPARK-5932][CORE] Use consistent naming for size properties (#5574)


In core/src/main/scala/org/apache/spark/SparkConf.scala<https://github.com/apache/spark/pull/5574#discussion_r29066890>:

> @@ -407,7 +474,13 @@ private[spark] object SparkConf extends Logging {
>          "The spark.cache.class property is no longer being used! Specify storage levels using " +
>          "the RDD.persist() method instead."),
>        DeprecatedConfig("spark.yarn.user.classpath.first", "1.3",
> -        "Please use spark.{driver,executor}.userClassPathFirst instead."))
> +        "Please use spark.{driver,executor}.userClassPathFirst instead."),
> +      DeprecatedConfig("spark.kryoserializer.buffer.mb", "1.4",


Since you added the alternate config, you can remove this entry.

—
Reply to this email directly or view it on GitHub<https://github.com/apache/spark/pull/5574/files#r29066890>.
________________________________________________________

The information contained in this e-mail is confidential and/or proprietary to Capital One and/or its affiliates. The information transmitted herewith is intended only for use by the individual or entity to which it is addressed.  If the reader of this message is not the intended recipient, you are hereby notified that any review, retransmission, dissemination, distribution, copying or other use of, or taking of any action in reliance upon this information is strictly prohibited. If you have received this communication in error, please contact the sender and delete the material from your computer.